### PR TITLE
[sdk]: 3.2.3 changelog and bump version for SDKs

### PR DESCRIPTION
**Pull request by Wayonb**

### CODE REVIEW

File: sdk/python/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
 
 ## next
 
+## [3.2.3] - 02-Dec-2024
+
+### Added
+- add toJson() to the generated model classes to get a JSON-safe representation of the model.
```
Comment (Jaguar0625) [inline sdk/python/CHANGELOG.md:11]: to_json

File: sdk/python/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,18 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
...
+- add extractSigningPayload() to facade to get signing payload which allows signing by hardware keys.
```
Comment (Jaguar0625) [inline sdk/python/CHANGELOG.md:12]: extract_signing_payload

File: sdk/javascript/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,19 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
...
+- add toJson() to the generated model classes to get a JSON-safe representation of the model.
+- add extractSigningPayload() to facade to get signing payload which allows signing by hardware keys.
+
+### Changed
+- export bytesToBigInt(), bytesToInt(), deepCompare(), intToBytes(), isHexString() and tryParseUint() from sdk via utils
+- (BREAKING NEM) NEM cosignature transaction multisig_transaction_hash field renamed to other_transaction_hash
+
+### Fixed
+- (NEM) add non-verifiable cosignature transaction to allow signing of the cosignature transaction.
+
```
Comment (Jaguar0625) [inline sdk/javascript/CHANGELOG.md:20]: add footer entry

File: sdk/python/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,18 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
...
+
+### Changed
+- (BREAKING NEM) NEM cosignature transaction multisig_transaction_hash field renamed to other_transaction_hash
+
+### Fixed
+- (NEM) add non-verifiable cosignature transaction to allow signing of the cosignature transaction.
+
```
Comment (Jaguar0625) [inline sdk/python/CHANGELOG.md:19]: add footer entry

File: sdk/javascript/CHANGELOG.md (hunk 3 above)
Comment (Wayonb) [reply sdk/javascript/CHANGELOG.md:20]: always for this 🤦🏾‍♂️

**Review by Jaguar0625 — APPROVED** [review]
👍

**Review by AnthonyLaw — APPROVED** [review]
👍🏼

**Pull request merged**